pkg/notifier/ocmnotifier.go

```diff
@@ -73,12 +64,16 @@ type ocmNotifier struct {
 
 func (s *ocmNotifier) NotifyState(value NotifyState, description string) error {
 
-	policyId, err := s.getPolicyIdForUpgradeConfig()
+	clusterId, err := s.getInternalClusterId()
```

Comment:
So the reason I initially put clusterID in the struct was to minimize the number of times a cluster has to reach out to OCM to get its cluster ID - do it once. This approach, if I'm reading it right, will result in a call every X minutes and makes the cluster very chatty with OCM.

Can we change this so that if it gets the cluster ID successfully once, it permanently caches it and doesn't attempt to re-get it (until the operator pod next restarts)?

Reply:
Ignore me - this is fine.